include/dali/c_api.h

```diff
@@ -341,6 +347,13 @@ DLL_PUBLIC void
 daliCopyTensorNTo(daliPipelineHandle *pipe_handle, void *dst, int n, device_type_t dst_type,
                   cudaStream_t stream, int non_blocking);
 
+
+/**
+ * @brief Gets direct access to the dali buffer containing an output of the pipeline
+ */
+DLL_PUBLIC void daliOutputPtr(daliPipelineHandle* pipe_handle, int n,
+                              const void **out_ptr, size_t *out_len);
```

Comment:
I think the `const void **out_ptr` signature is unlucky, cause this won't work:
```c++
void fn(const void ** ptr) {}

void *ptr;
fn (&ptr);   // illegal void** -> const void**
```
Shouldn't it be just a `void ** out_ptr`? I wonder what benefits there are of having this `const` here?

Reply:
It's a pointer to const data, we don't want the memory to be modified, that's why the const